themes/default/content/docs/guides/adopting/from_kubernetes.md

```diff
@@ -557,6 +565,202 @@ $ curl http://$(pulumi stack output frontendIp)
+export const frontendIp = svc.status.loadBalancer.ingress[0].ip;
+```
+
+{{% /choosable %}}
+
+{{% choosable language python %}}
+
+```python
+import pulumi
+from pulumi import Output
+from pulumi_kubernetes.core.v1 import Service
+from pulumi_kubernetes.helm.v3 import Release, ReleaseArgs, RepositoryOptsArgs
+
+# Deploy the bitnami/wordpress chart.
+wordpress = Release(
+    "wpdev",
+    ReleaseArgs(
+        chart="wordpress",
+        repository_opts=RepositoryOptsArgs(
+            repo="https://charts.bitnami.com/bitnami",
+        ),
+        # Force to use ClusterIP so no assumptions on support for LBs etc. is required.
+        version="9.6.0",
+        values={
+            "service": {
+                "type": "ClusterIP",
+            }
+        },
+    ),
+)
+
+srv = Service.get("wpdev-wordpress", Output.concat(wordpress.status.namespace, "/", wordpress.status.name, "-wordpress"))
+# Export the ingress IP for Wordpress frontend.
+pulumi.export("frontendIP", srv.status.load_balancer.ingress[0].ip)
+```
+
+{{% /choosable %}}
+
+{{% choosable language go %}}
+
+```go
+package main
+
+import (
+	"fmt"
+	corev1 "github.com/pulumi/pulumi-kubernetes/sdk/v3/go/kubernetes/core/v1"
+	"github.com/pulumi/pulumi-kubernetes/sdk/v3/go/kubernetes/helm/v3"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+)
+
+func main() {
+	pulumi.Run(func(ctx *pulumi.Context) error {
+		// Deploy the bitnami/wordpress chart.
+		wordpress, err := helm.NewRelease(ctx, "wpdev", &helm.ReleaseArgs{
+			Version: pulumi.String("9.6.0"),
+			Chart:   pulumi.String("wordpress"),
+			RepositoryOpts: &helm.RepositoryOptsArgs{
+				Repo: pulumi.String("https://charts.bitnami.com/bitnami"),
+			},
+		})
+
+        // Export the ingress IP for Wordpress frontend.
+		frontendIp := pulumi.All(wordpress.Status.Namespace(), wordpress.Status.Name()).ApplyT(func(r interface{})(interface{}, error){
+			arr := r.([]interface{})
+			namespace := arr[0].(*string)
+			name := arr[1].(*string)
+			svc, err := corev1.GetService(ctx, "svc", pulumi.ID(fmt.Sprintf("%s/%s-wordpress", *namespace, *name)), nil)
+			if err != nil {
+				return "", nil
+			}
+			return svc.Status.LoadBalancer().Ingress().Index(pulumi.Int(0)).Ip(), nil
+
+		})
+		ctx.Export("frontendIp", frontendIp)
+
+		if err != nil {
+			return err
+		}
+
+		return nil
+	})
+}
+```
+
+{{% /choosable %}}
+
+{{% choosable language csharp %}}
+
+```csharp
+using Pulumi;
+using Pulumi.Kubernetes.Core.V1;
+using System.Collections.Generic;
+using Pulumi.Kubernetes.Types.Inputs.Helm.V3;
+using Pulumi.Kubernetes.Helm.V3;
+
+class MyStack : Stack
+{
+    public MyStack()
+    {
+        // Deploy the bitnami/wordpress chart.
+        var wordpress = new Release("wpdev", new ReleaseArgs
+        {
+            Chart = "wordpress",
+            RepositoryOpts = new RepositoryOptsArgs
+            {
+                Repo = "https://charts.bitnami.com/bitnami"
+            },
+            Version = "9.6.0",
+        });
```

Comment:
This version doesn't include the ClusterIP config

Reply:
Removed from the python version instead.